React/game/src/app/components/Screen/Game/styles.scss

```diff
@@ -0,0 +1,18 @@
+body {
```

Comment:
body is a globla style, sou must move it to `scss/index.scss` file

Reply:
👍